sdk/core/azure-core/inc/azure/core/context.hpp

```diff
@@ -205,6 +206,7 @@ namespace Azure { namespace Core {
      */
     template <class T> bool TryGetValue(Key const& key, T& outputValue) const
     {
+      // LCOV_EXCL_START
       for (auto ptr = m_contextSharedState; ptr; ptr = ptr->Parent)
```

Comment:
This we should be testing, imo. It is part of correct usage of `TryGetValue`, no? Why are we skipping it?